src/Validator/ErrorElement.php

```diff
@@ -149,7 +149,7 @@ public function getSubject()
     /**
      * @param string|array $message
      * @param array        $parameters
-     * @param null         $value
+     * @param mixed|null   $value
```

Comment:
I think the 3 last phpdocs should simply be removed

Reply:
Not part of this PR. Cleanup should be done afterwards